src/format-files.js

```
@@ -89,8 +89,22 @@ function formatFilesFromArgv({
 }
 
 async function formatStdin(prettierESLintOptions) {
-  const stdin = await getStdin();
-  const stdinValue = stdin.trim();
+  let stdinValue = '';
+
+  if (!process.stdin.isTTY) {
+    try {
+      const stdin = await text(process.stdin);
+      stdinValue = stdin.trim();
+    } catch (error) {
```

Why should we `try/catch` here? Why not just let it throw?

---

You're right. We don't need that!